scripts/statesync.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# print every command
+set -ux
+
+rm -fr ~/.ununifi
+
+# make sure ununifid version is over v1.0.0-beta.3.1
+# git clone https://github.com/UnUniFi/chain.git ununifi
+# cd ununifi
+# git checkout v1.0.0-beta.3.1
+# make install
+
+ununifid init test
+
+wget -O ~/.ununifi/config/genesis.json https://raw.githubusercontent.com/UnUniFi/network/main/launch/ununifi-beta-v1/genesis.json
+
+INTERVAL=1000
+
+# get trust hash and trust height
+
+LATEST_HEIGHT=$(ununifid q block --node https://a.lcd.ununifi.cauchye.net:443 | jq -r .block.header.height)
+BLOCK_HEIGHT=$(($LATEST_HEIGHT-$INTERVAL))
+TRUST_HASH=$(ununifid q block $BLOCK_HEIGHT --node https://a.lcd.ununifi.cauchye.net:443 | jq -r .block_id.hash)
```

Comment:
define TRUST_NODE

Reply:
fixed it at https://github.com/UnUniFi/chain/pull/261/commits/35d294baa566465bdb7c59fb306f58aaea96204f.
Please check it.